README.md

```diff
@@ -1 +1,43 @@
-# contributor-website-v2
\ No newline at end of file
+# contributor-website-v2
+Contributors website is a one-stop-shop for anyone associated with moja global to get a glimpse of our projects and their working. The website also holds blogs and guides by the moja family to help you better understand our work and hopefully learn from others experience.
+
+## What is moja global?
+[*moja global*](http://moja.global/) is a collaboration under the [Linux Foundation](https://linuxfoundation.org/) that aims for the widest possible collaboration on and use of credible tools to better manage the land sector. The flagship software is the [Full Lands Integration Tool (FLINT)](https://github.com/moja-global/flint), a tool to estimate emissions and sinks of greenhouse gasses from forestry and agriculture.  
+
+## Development Setup
+Before setting up the project make sure the LTS (Long Term Support) version of NodeJs is installed with a package manager of your choice. We recommend NPM or Yarn.
+
+1. Fork the repository.
+
+2. Clone the repository.
+```
+https://github.com/username/contributor-website-v2.git
+```
+
+3. Go into the directory containing the project.
+```
+cd contributor-website-v2/website
+```
+
+4. Install all the dependencies
+```
+npm install
+```
+
+5. Start the development server.
+```
+npm run start
+```
+
+## How to Get Involved?  
+
+moja global welcomes a wide range of contributions as explained in [Contributing document](https://github.com/moja-global/About-moja-global/blob/master/CONTRIBUTING.md) and in the [About moja-global Wiki](https://github.com/moja-global/.github/wiki).  
+
+## FAQ and Other Questions  
+
+* You can find FAQs on the [Wiki](https://github.com/moja.global/.github/wiki).  
```

Comment:
The link here is dead, I guess

Reply:
Will check. If that's the case, I think we need to change it in the About moja global repo too.